cmd/storj-sim/network.go

```diff
@@ -278,25 +279,19 @@ func newNetwork(flags *Flags) (*Processes, error) {
 		}
 	}
 
-	// Create the API process for each satellite
-	var satelliteAPIs []*Process
+	// Create the peer process for each satellite API
 	for i, satellite := range satellites {
 		process := processes.New(Info{
-			Name:       fmt.Sprintf("satellite-api/%d", i),
+			Name:       fmt.Sprintf("satellite-peer/%d", i),
 			Executable: "satellite",
 			Directory:  filepath.Join(processes.Directory, "satellite", fmt.Sprint(i)),
-			Address:    net.JoinHostPort(host, port(satelliteAPI, i, publicGRPC)),
+			Address:    "",
 		})
-		satelliteAPIs = append(satelliteAPIs, process)
 
 		process.Arguments = withCommon(process.Directory, Arguments{
 			"run": {
-				"api",
-				"--console.address", net.JoinHostPort(host, port(satelliteAPI, i, publicHTTP)),
-				"--marketing.address", net.JoinHostPort(host, port(satelliteAPI, i, privateHTTP)),
 				"--server.address", process.Address,
```

Comment:
Why does it need the server.address argument?

Reply:
oh, i guess we don't need it since there isn't this line ```readConfigString(&process.Address, process.Directory, "server.address")```

Reply:
yeah, just yesterday when i was debugging things I needed to add that at one point, but many things have changed since then and you are right, its unnecessary now